### PR TITLE
nodejs-ca: Update Get Involved

### PR DESCRIPTION
**Pull request by Tsuna-mi**
This PR add Nodeiflux to Get Involved to catalan version. 
All updates follow English file version.
To review it we need some help from native catalan speakers. Thanks!

(cc @kvhuaxa @franher @axarrafols @jose-lara)
Related with #1766

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
English part is well, however I'm not familiar with ca……Maybe this need another one to check with :)

**Comment by Tsuna-mi** [comment]
I'll try to get colleagues help this week ;)

File: locale/ca/get-involved/index.md [hunk 1]
```diff
@@ -12,20 +12,21 @@ contribuir de la manera que puguin. Si vols [reportar un error](https://github.c
 
 ## Discussió de la comunitat
 
-- La [llista d'errors en ](https://github.com/nodejs/node/issues) és el lloc per discutir les característiques del nucli de Node.js.
+- La [llista Github de qüestions](https://github.com/nodejs/node/issues) és el lloc per discutir les característiques del nucli de Node.js.
```
Comment (luisprieto) [inline locale/ca/get-involved/index.md]: I find "incidència" a better translation of "issue", so I'd put here:
`La [llista d'incidències de Github]`

File: locale/ca/get-involved/index.md [hunk 2]
```diff
@@ -12,20 +12,21 @@ contribuir de la manera que puguin. Si vols [reportar un error](https://github.c
...
 - La [Community Committee](https://github.com/nodejs/community-committee) és una comissió de primer nivell a la Fundació Node.js centrada en els esforços que afronta la comunitat.
 
-
 ## Aprenentatge
 
 - La [Documentació oficial de l'API](/api) detalla l'API de Node.
 - [NodeSchool.io](http://nodeschool.io) li ensenyarà conceptes de Node.js de forma interactiva mitjançant jocs utilitzant la línia de comandes.
 - L'[etiqueta de Node.js en Stack Overflow](http://stackoverflow.com/questions/tagged/node.js) col·lecciona nova informació cada dia.
+- L'[etiqueta de Node.js en la DEV Community](https://dev.to/t/node) és un lloc on compartir projectes de Node.js, articles i tutorials, així com iniciar debats i demanar realimentació sobre temes relacionats amb Node.js. Els desenvolupadors de tots els nivells d'experiència són benvinguts a participar.
+- [Nodeiflux](https://discordapp.com/invite/vUsrbjd) és una comunitat amigable de desenvolupadors de backend amb Node que es recolzen mútuament en Discord.
```
Comment (luisprieto) [inline locale/ca/get-involved/index.md]: `... es recolzen mútuament a Discord.`

**Review by luisprieto — CHANGES_REQUESTED** [review]

**Review by luisprieto — APPROVED** [review]

File: locale/ca/get-involved/index.md [hunk 3]
```diff
@@ -12,20 +12,21 @@ contribuir de la manera que puguin. Si vols [reportar un error](https://github.c
...
 - La [Community Committee](https://github.com/nodejs/community-committee) és una comissió de primer nivell a la Fundació Node.js centrada en els esforços que afronta la comunitat.
 
-
 ## Aprenentatge
 
 - La [Documentació oficial de l'API](/api) detalla l'API de Node.
 - [NodeSchool.io](http://nodeschool.io) li ensenyarà conceptes de Node.js de forma interactiva mitjançant jocs utilitzant la línia de comandes.
 - L'[etiqueta de Node.js en Stack Overflow](http://stackoverflow.com/questions/tagged/node.js) col·lecciona nova informació cada dia.
+- L'[etiqueta de Node.js en la DEV Community](https://dev.to/t/node) és un lloc on compartir projectes de Node.js, articles i tutorials, així com iniciar debats i demanar realimentació sobre temes relacionats amb Node.js. Els desenvolupadors de tots els nivells d'experiència són benvinguts a participar.
+- [Nodeiflux](https://discordapp.com/invite/vUsrbjd) és una comunitat amigable de desenvolupadors de backend amb Node que es recolzen mútuament a Discord.
```
Comment (franher) [inline locale/ca/get-involved/index.md]: `de backend amb Node.js`

I think we should use always the same name for Node.js :)

File: locale/ca/get-involved/index.md [hunk 4]
```diff
@@ -40,6 +41,6 @@ contribuir de la manera que puguin. Si vols [reportar un error](https://github.c
 - [Grup d'usuaris de Japó](http://nodejs.jp/)
 - [Comunitat Coreana de Node.js](http://nodejs.github.io/nodejs-ko/)
 - [Comunitat de Nicaragua de Node.js](http://nodenica.com/)
-- [Grup de Facebook de llengua castellana per a Node.js](https://www.facebook.com/groups/node.es/)
-- [Comunitat de llenguatge hispà](http://nodehispano.com)
+- [Grup de Facebook en espanyol per Node.js](https://www.facebook.com/groups/node.es/)
```
Comment (franher) [inline locale/ca/get-involved/index.md]: `Grup de Facebook en espanyol de Node.js` I find `de` better than `per`here.

**Review by franher — COMMENTED** [review]
@Tsuna-mi thank you for the contribution 

I did a couple of suggestions :)

File: locale/ca/get-involved/index.md (hunk 3 above)
Comment (Tsuna-mi) [reply locale/ca/get-involved/index.md]: I just translate from the original document in english... where is approved/published just with 'Node'
I am agree with you and consistency using Node.js. 

More opinions?

**Review by franher — APPROVED** [review]

File: locale/ca/get-involved/index.md (hunk 3 above)
Comment (fhemberger) [reply locale/ca/get-involved/index.md]: Yes, please use "Node.js". Should be fixed in the English version as well. ;)

**Comment by Tsuna-mi** [comment]
PR ready to merge
Thanks for your review @luisprieto and @franher  :)

**Comment by fhemberger** [comment]
Thanks everyone!

**Pull request merged**